src/app/forms/custom-sections/psv-brakes/psv-brakes.component.html

```diff
@@ -0,0 +1,106 @@
+<app-switchable-input
+  [form]="brakesForm"
+  [type]="editTypes.AUTOCOMPLETE"
+  name="brakeCode"
+  label="Brake code"
+  [isEditing]="isEditing"
+  [width]="widths.L"
+  [options]="(brakeCodeOptions$ | async) || []"
+></app-switchable-input>
+
+<app-switchable-input
+  [form]="brakesForm"
+  [type]="editTypes.TEXT"
+  name="dataTrBrakeOne"
+  label="Service *"
+  [isEditing]="isEditing"
+  [width]="widths.XL"
+></app-switchable-input>
+
+<app-switchable-input
+  [form]="brakesForm"
+  [type]="editTypes.TEXT"
+  name="dataTrBrakeTwo"
+  label="Secondary *"
+  [isEditing]="isEditing"
+  [width]="widths.XL"
+></app-switchable-input>
+
+<app-switchable-input
+  [form]="brakesForm"
+  [type]="editTypes.TEXT"
+  name="dataTrBrakeThree"
+  label="Parking *"
+  [isEditing]="isEditing"
+  [width]="widths.XL"
+></app-switchable-input>
+
+<div class="row">
+  <app-switchable-input
+    [form]="brakesForm"
+    [type]="editTypes.DROPDOWN"
+    name="retarderBrakeOne"
+    label="Retarder 1"
+    [isEditing]="isEditing"
+    [options]="retarderOptions"
+  ></app-switchable-input>
+
+  <app-switchable-input
+    [form]="brakesForm"
+    [type]="editTypes.DROPDOWN"
+    name="retarderBrakeTwo"
+    label="Retarder 2"
+    [isEditing]="isEditing"
+    [options]="retarderOptions"
+  ></app-switchable-input>
+</div>
+
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header"></th>
+      <th scope="col" class="govuk-table__header">No wheels locked</th>
+      <th scope="col" class="govuk-table__header">Up to and including half wheels locked</th>
+    </tr>
+  </thead>
+
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">Service</th>
+      <td class="govuk-table__cell">{{ ((vehicleTechRecord.grossLadenWeight || 0) * 16) / 100 }}</td>
```

Comment:
Numbers need to be rounded to nearest whole number

Reply:
Done